services/web/client/source/class/qxapp/desktop/StudyEditor.js

```diff
@@ -614,6 +615,13 @@ qx.Class.define("qxapp.desktop.StudyEditor", {
 
     __attachEventHandlers: function() {
       this.__blocker.addListener("tap", this.__sidePanel.toggleCollapsed.bind(this.__sidePanel));
+
+      qx.event.message.Bus.getInstance().subscribe("maximizeIframe", msg => {
```

Comment:
all instances will react ... is that intended ?

Reply:
space is great, my only concern is that the bus is global, any iframe maximizing will advertise this to the bus ... all elements will listen to it, regardless if they are currently on screen or not ... could will trigger lots of code which would not be necessary.

Reply:
Corrected!